### DISCUSSION TITLE
Confused with camera width and height setting on config - 320x320 or 1280x720?

### DISCUSSION THREAD

**Discussion by Yonny24** [blakeblackshear/frigate#7238] [Ask A Question] [answered]
I understand that frigate resizes to 320x320 for analysis which uses cpu.
If I set my mainstream camera to 
    detect:
      width: 320
      height: 320
unfortunately I get no events but I can see myself being detected. Possibly false positives? No filters added.

But if I reset to 
      detect:
      width: 1280 
      height: 720 
      
I get true positives and events created.

So if it still using that 320x320 resized for both scenarios why do I get better detection on the 1280x720? Is the quality also reduced if I specify a 320x320 in the config?

Thanks

**Comment by NickM-27** [comment]
320x320 is the size of the resized region that is sent for detection, frigate does not send the entire camera frame for detection.

**Reply by Yonny24** [reply]
Yes I realise that so if my entire camera frame is set at 320x320 in the config then it analysis the whole frame. I see the green box around the entire debug frame.

So this 320x320 camera frame - entire size will result in poorer detections?

The 1280x720, even though only that 320x320 section is sent for analyses will result in more accurate detections?
That is what I observed comparing the 2 configs.

**Reply by NickM-27** [reply]
Yes, analyzing the whole frame leads to worse results in detection

**Reply by Yonny24** [reply, marked as answer]
Ah that explains it thanks.
